pgcli/packages/sqlcompletion.py

```diff
@@ -42,6 +42,54 @@
 Path = namedtuple('Path', [])
 
 
+class SqlDocument(object):
```

Comment:
I started with Document because `full_text`/`text_before_cursor` could contain multiple `;` separated statements. But then the constructor parses the document and throws alway all but the current statement so Statement does seem more accurate.